packages/code-du-travail-frontend/src/common/AccessibilityModal.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import {
+  Heading,
+  InputCheckbox,
+  Modal,
+  Title,
+  theme
+} from "@socialgouv/react-ui";
+
+import { useTheme } from "../layout/ThemeProvider.js";
+
+export const AccessibilityModal = props => {
+  const [isModalOpened, setModalOpen] = useState(false);
+  const { currentTheme, toggleTheme } = useTheme();
+  return (
+    <>
+      {React.cloneElement(props.children, {
```

Comment:
It not understood why you need to clone children in the first time. 
A render render prop approach would be simpler, less magic, and less error prone what if I do 
```
<AccessibilityModal onClick={trackAccessibility}>
   <Button>texte dans le bouton</Button>
</AccessibilityModal>
```

Reply:
To be honest I copy paster the code from `ServiceDeRenseignementsModal`  and after taking a look at it, we do have a `renderProp` usage here:
```
                    <ConventionModal key="convention-modal">
                      {openModal => (
                        <Button variant="secondary" onClick={openModal}>
                          Chercher une convention collective
                        </Button>
                      )}
```
I'm going to refactor both if that's ok for you ?

Reply:
yup !